### PR TITLE
ACS-4966 Add path to Category model

### PR DESCRIPTION
**Pull request by MichalKinas**
https://alfresco.atlassian.net/browse/ACS-4966
This PR is about adding `path` param to Categories API.

### CODE REVIEW

**Comment by tpage-alfresco** [comment]
Sorry - I added people prematurely to this PR. @MichalKinas You haven't added a description about the new `include=path` option to the category endpoints.

**Comment by tpage-alfresco** [comment]
Sorry - I added people prematurely to this PR.

@MichalKinas You haven't added a description about the new `include=path` option to the category endpoints.

**Comment by MichalKinas** [comment]
> Sorry - I added people prematurely to this PR. @MichalKinas You haven't added a description about the new `include=path` option to the category endpoints.

Right thank you, added now

**Comment by azakrzewski-hy** [comment]
> > Sorry - I added people prematurely to this PR. @MichalKinas You haven't added a description about the new `include=path` option to the category endpoints.
> 
> Right thank you, added now

Should we keep * path field in both descriptions: for categoryEntryIncludeParam and categoryEntryIncludePathParam ?

**Comment by MichalKinas** [comment]
> > > Sorry - I added people prematurely to this PR. @MichalKinas You haven't added a description about the new `include=path` option to the category endpoints.
> > 
> > 
> > Right thank you, added now
> 
> Should we keep * path field in both descriptions: for categoryEntryIncludeParam and categoryEntryIncludePathParam ?

Yes `categoryEntryIncludeParam` is used when both `count` and `path` options are available and `categoryEntryIncludePathParam` is used when only `path` option is available for given endpoint

File: src/main/webapp/definitions/alfresco-core.yaml [hunk 1]
```diff
@@ -228,6 +228,18 @@ parameters:
     description: |
       Returns additional information about the category. The following optional fields can be requested:
       * count
+      * path
+    required: false
+    type: array
+    items:
+      type: string
+    collectionFormat: csv
+  categoryEntryIncludePathParam:
```
Comment (tpage-alfresco) [inline src/main/webapp/definitions/alfresco-core.yaml:237]: It's a little confusing to have two of these, but we should fix it as part of https://alfresco.atlassian.net/browse/ACS-4923

**Comment by tpage-alfresco** [comment]
I think you're still missing some endpoints in this change. I think you added `include=path` to GET category, PUT category, GET subcategories, POST subcategories, GET nodes/{id}/categories and POST nodes/{id}/categories in the other PR.

**Comment by MichalKinas** [comment]
> I think you're still missing some endpoints in this change. I think you added `include=path` to GET category, PUT category, GET subcategories, POST subcategories, GET nodes/{id}/categories and POST nodes/{id}/categories in the other PR.

@tpage-alfresco are you sure? I've just double checked and each Category related endpoint except DELETE has `include=path` added in its description.

**Review by tpage-alfresco — APPROVED** [review]
No, you're right - this looks fine. Sorry for the confusion!

**Pull request merged**